python/test/test_polars.py

```diff
@@ -369,7 +369,7 @@ def test_polars_describe():
             "column": ["len", "A"],
             "aggregate": ["Len", "Sum"],
             "distribution": ["Integer Laplace", "Integer Laplace"],
-            "scale": [8.0, 8.0],
+            "scale": [4.0, 12.0],
```

Comment:
Can you add a comment explain the 1/3 ratio? I see this in the original issue, and I can imagine that sum is more sensitive than len, but I don't immediately see where that ratio is coming from.